scripts/create-data.js

```diff
@@ -1164,14 +1178,13 @@ const createUserAndColonyData = async () => {
       }`,
     );
 
-    // verify users
+    // Add verified members to colony
```

Comment:
There's a issue with this, as the first three 3 dev accounts are no longer verified  with this configuration

Please account for them as well